apiserver/facades/client/charmhub/convert.go

```diff
@@ -142,32 +146,44 @@ func transformInfoChannelMap(channelMap []transport.InfoChannelMap) ([]string, m
 	return trackList, channels
 }
 
-func convertPlatforms(in []transport.Platform) []params.Platform {
+// convertBasesToPlatforms converts a base to a platform.  Is mean to be a short
+// term solution due to schedule.  It should be removed once platforms are
+// replaced with bases through out the code.
+func convertBasesToPlatforms(in []transport.Base, isKub bool) []params.Platform {
 	out := make([]params.Platform, len(in))
 	for i, v := range in {
+		channel := v.Channel
+		if isKub {
```

Comment:
Is the direction mixed up here? We *do* want to store series "kubernetes" in this case right?
```go
	for i, v := range in {
		var series string
		if isKub {
			series = "kubernetes"
		} else {
			series, _ = coreseries.VersionSeries(v.Channel)
		}
		out[i] = params.Platform{
			Architecture: v.Architecture,
			OS:           v.Name,
			Series:       series,
		}
	}
```

Reply:
Yes, thank you. And I need to use the series, to get the OS, or we'll send up with something like Ubuntu Kubernetes.